README.md

```diff
@@ -25,24 +25,24 @@ Installation
 
 The best way to run Puppet is with [Puppet Enterprise](https://puppetlabs.com/puppet/puppet-enterprise),
 which also includes orchestration features, a web console, and professional support.
-[The PE documentation is available here.](https://docs.puppetlabs.com/pe/latest)
+The PE documentation is available [here](https://puppetlabs.com/docs/pe/latest).
 
 To install an open source release of Puppet,
-[see the installation guide on the docs site.](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html)
+see the [Installation Guide](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html) on the docs site.
```

Comment:
This link got missed, should be updated to `https://puppet.com/docs/puppet/latest/install_pre.html`